app.docker

```diff
@@ -10,4 +12,6 @@ RUN apt-get update \
 #  && docker-php-ext-configure mcrypt \
   && curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
 
-WORKDIR /var/www
+ADD . /var/www
+
+RUN chown -R www-data:www-data /var/www
```

Comment:
This makes directory /var/www owned by www-data which is the default user for php-fpm.